spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require 'codeclimate-test-reporter'
 require 'factory_girl_rails'
 require 'simplecov'
+require 'rails_helper'
```

Comment:
Changing the load order - interesting fix. So does this mean that all of our tests should

```ruby
require 'spec_helper'
```

at the top now?